site/docs/selection/clear.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Just like `on` that is listed  here: https://vega.github.io/vega-lite/docs/selection.html#selection-properties, I think clear is a basic selection property so should be listed in the same section instead of having its own page.

Reply:
To be fair, I think our  line between what is selection transform and what is not isn't very clear. 

However, given that "clear" should be paired with "on", it's  better to put them in the same place for now.

Reply:
Agreed, the line between a selection property and transformation is unclear. Nevertheless, my preference would be to keep clear on a separate page for a couple of reasons:

(1) It would mirror `toggle` that lives on its own separate page.

(2) The main selection documentation page is already pretty long. `clear` has some non-trivial explanation associated with it plus 1-2 examples. We could shoe-horn it under the `Selection Properties` heading, but seems unnecessary given that it is well-scoped to be standalone (whereas `on` is a little more core to a selection's definition). 

(Fwiw, I had planned on moving `init` out to a separate page too, because of how complex its definition can be. But I ran out of time when I was developing that feature.)

We can water down the "transformation" language and just call it a "property" if you prefer. This is what we do with "bind" (but not for any principled reason). In fact, I'm not really sure that we need to explicitly distinguish between property/transformation in the documentation -- I think that distinction is more useful in the research context.

Reply:
Sounds good. :) 

If we say they are all selection properties, I agree that we should have clear on a separate page as the main page is already too long. 

In fact, in a separate  PR later, we should just make a page for each selection "property".  I also wonder if on should be on the same page as "clear", just like the width/height page.   But I think we can handle that later in a separate PR too.